OAuth/ResourceOwner/FiwareResourceOwner.php

```diff
@@ -86,7 +85,7 @@ public function getUserInformation(array $accessToken, array $extraParameters =
         }
 
         $response = $this->getUserResponse();
-        $response->setData($content->getContent());
+        $response->setData((string) $content->getBody());
```

Comment:
I see what's going on, I have a project containing an old version of Guzzle: https://github.com/guzzle/guzzle/blob/v3.8.1/src/Guzzle/Http/Message/Response.php#L187